packages/ui/src/lib/settingsNavItem/SettingsNavItem.svelte

```diff
@@ -67,18 +69,20 @@ function click(): void {
     class:hover:text-[color:var(--pd-secondary-nav-text-hover)]={!selected}
     class:hover:bg-[var(--pd-secondary-nav-text-hover-bg)]={!selected}
     class:hover:border-[var(--pd-secondary-nav-text-hover-bg)]={!selected}>
-    <span class="group-hover:block flex flex-row items-center" class:capitalize={!child}>
+    <span
+      class="group-hover:block flex gap-x-4 items-center"
+      class:flex-row={iconPosition === 'left'}
+      class:flex-row-reverse={iconPosition === 'right'}
+      class:capitalize={!child}>
```

Comment:
Logic is to reverse the row, since we have only the text and the icon.
Moreover replacing the `margin right` by a `gap` so it works in both scenarios